perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_defaultDataQuota.java

```diff
@@ -119,7 +91,7 @@ public void checkAttributeSemantics(PerunSessionImpl perunSession, Resource reso
 		//Compare DefaultDataQuota with DefaultDataLimit
 		if (quotaNumber.compareTo(BigDecimal.valueOf(0)) == 0) {
 			if (limitNumber.compareTo(BigDecimal.valueOf(0)) != 0) {
-				throw new WrongReferenceAttributeValueException(attribute, attrDefaultDataLimit, resource, null, resource, null, "Try to set unlimited quota, but limit is still " + defaultDataLimitNumber + defaultDataLimitLetter);
+				throw new WrongReferenceAttributeValueException(attribute, attrDefaultDataLimit, resource, null, resource, null, "Try to set unlimited quota, but limit is still " + limitNumber + defaultDataLimitLetter);
 			}
 		} else if (limitNumber.compareTo(BigDecimal.valueOf(0)) != 0 && defaultDataLimitLetter != null && defaultDataQuotaLetter != null) {
```

Comment:
I think you can erase the check if defaultDatalimitLetter is null as you did in the defaultDataLimit module. Moreover, there is the same question on defaultDataQuotaLetter as in the previous module.